### DISCUSSION TITLE
Autoscaling

### DISCUSSION THREAD

**Discussion by Youngwoo-git** [kubeflow/kfserving#2217] [General]
/kind feature

Currently, I'm using the dockerfile provided in this repository. Somehow I managed to run the server, yet it seems to fail in autoscale. Would you mind guiding me to the document to customize the autoscaling option if there is any?

**Comment by Suresh-Nakkeran** [comment]
@Youngwoo-git Can you refer the [autoscale](https://kserve.github.io/website/0.8/modelserving/autoscaling/autoscaling/) document?
